composer.json

```diff
@@ -22,7 +22,8 @@
     "php": ">=7.2",
     "ruflin/elastica": "^7.0",
     "nette/di": "^3.0",
-    "nette/utils": "^3.0"
+    "nette/utils": "^3.0",
+    "nette/schema": "^1.2"
```

Comment:
nette/schema is tight with nette/di, we can skip it here.

Reply:
As I wrote in the first post:

> Package `nette/schema` is required because of usage of the method `::skipDefaults()` that has been added in the version `^1.2`.

All versions of `nette/di` requires `nette/schema` in versions `^1.0` or `^1.1` so here can be compatibility problem.
Another solution could be to remove `nette/schema` from requirements and add the section `conflict`:

```json
{
	"conflict": {
		"nette/schema": "<1.2"
	}
}
```